docs/content/gs-adding-custom-3scale-routes.adoc

```diff
@@ -0,0 +1,59 @@
+[id='gs-adding-custom-3scale-routes']
+
+ifdef::env-github[]
+:imagesdir: ../images/
+endif::[]
+
+= Allow using public hosts for 3scale APIs in {PRODUCT}
+
+Customers might want to use public custom base URLs for their 3scale APIs. In order to allow that an OpenShift route has to be added to the 3scale namespace first.
+
+NOTE: The `customer-admin` user has the permissions required for the following steps.
+
+== Creating the OpenShift route
+
+A new OpenShift route for the public has to be created.
+
+First make sure you are logged in as `customer-admin`
+
+```sh
+$ oc login -u customer-admin
+```
+
+Create a file named `custom-route.yaml` with the following contents:
+
+```yaml
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: <route name>
+  namespace: openshift-3scale
+spec:
+  host: <route host>
+  port:
+    targetPort: gateway
+  tls:
+    insecureEdgeTerminationPolicy: None
+    termination: edge
+  to:
+    kind: Service
+    name: apicast-staging
+    weight: 100
+  wildcardPolicy: None
+```
+
+Fill in `<route name>` and `<route host>` in line 4 and 7 then create the route:
+
+```sh
+$ oc create -f ./custom-route.yaml
+```
+
+NOTE: once a route has been created it can't be deleted by `customer-admin`. This is to prevent accidental deletion of the auto generated routes.
```

Comment:
```suggestion
NOTE: You cannot delete any routes created using this method.```
1. I wonder if this needs to be at top of process
2. I don't think reason is appropriate here, but would make a good known issue in release notes